scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -462,12 +462,14 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 }
             }
 
+            let minTansaUrgency = $scope.config.tansa && $scope.config.tansa.urgency || 5;
```

Comment:
Should we put `5` at the top in `const DEFAULT_URGENCY = 5` or something? So it's easy to spot and config if needed.